src/main/java/io/rocketpartners/cloud/action/dynamo/DynamoDb.java

```diff
@@ -74,7 +74,7 @@
     */
    protected String       blueprintRow;
 
-   protected int          batchMax     = 25;
+   protected int          batchMax     = 20;
```

Comment:
Obtained the data to post to the dynamoDB by querying 'northwind/sql/orders'. But it has only 25 documents. So I tried the multiple batch write by editing the batch size to 20. So it will write 20 first then remaining 5